LiteCore/RevTrees/RevTree.hh

```diff
@@ -57,7 +57,7 @@ namespace litecore {
             kNew            = 0x04, /**< Has this rev been inserted since decoding? */
             kHasAttachments = 0x08, /**< Does this rev's body contain attachments? */
             kKeepBody       = 0x10, /**< Body will not be discarded after I'm a non-leaf */
-            kIsConflict     = 0x20, /**< Unresolved conflicting revision; should never be current */
+            kIsConflict      = 0x20, /**< Unresolved conflicting revision; should never be current */
```

Comment:
Looks like a mistake, a stray space character?

Reply:
I was trying out font ligatures and forgot about it. Changed it back.